modules/lb/elb.tf

```diff
@@ -8,6 +8,10 @@ resource "aws_lb" "ptfe" {
   ]
 
   idle_timeout = 3600 ## for ssh
+
+  tags = {
+    Name = "${var.prefix}"
+  }
 }
 
 resource "aws_lb_target_group" "https" {
```

Comment:
do we want to change these to have the prefix as well instead of a randomly generated one from terraform?

Reply:
Added it to all the resources I could that had an attribute of 'Name' or allowed 'tags'. Some resources didn't appear to support either.

Reply:
ok, it looked like these did support the name parameter which was why I asked https://www.terraform.io/docs/providers/aws/r/lb_target_group.html